### PR TITLE
user Settings page 

### PR DESCRIPTION
**Pull request by DIWAKARKASHYAP**
#92 

Suggested core feature

1- New page for user settings accessible via the user menu that currently only includes "sign out"

image

### CODE REVIEW

File: src/pages/project/userSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import Header from "@/src/components/layouts/header";
```
Comment (maxdeichmann) [inline src/pages/project/userSettings.tsx:1]: Right now, i get a 404 error when i hit the name of the user on the bottom left. Can you fix that?

File: src/components/layouts/layout.tsx [hunk 2]
```diff
@@ -361,57 +351,32 @@ export default function Layout(props: PropsWithChildren) {
...
-                    <Menu.Button className="flex w-full items-center gap-x-4 p-1.5 px-6 py-3 text-sm font-semibold leading-6 text-gray-900 hover:bg-gray-50">
-                      <span className="sr-only">Open user menu</span>
-                      <Avatar className="h-8 w-8">
-                        <AvatarImage
-                          src={session.data?.user?.image ?? undefined}
-                        />
-                        <AvatarFallback>
-                          {session.data?.user?.name
-                            ? session.data.user.name
+                    <Link href='./userSettings'>
```
Comment (maxdeichmann) [inline src/components/layouts/layout.tsx:354]: Right now, i get a 404 error when i hit the name of the user on the bottom left. Can you fix that?

File: src/pages/project/userSettings.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                : null}
+            </AvatarFallback>
+          </Avatar>
+        </div>
+        <ul
+          role="list"
+          className="mt-6 divide-y divide-gray-200 border-b border-t border-gray-200"
+        >
+          {instructionItems.map((item, itemIdx) => (
+            <li key={itemIdx}>
```
Comment (maxdeichmann) [inline src/pages/project/userSettings.tsx:54]: The UI here looks a bit large to me. Can you make sizings similar to the sizes on the left side bar?

File: src/components/layouts/layout.tsx (hunk 2 above)
Comment (DIWAKARKASHYAP) [reply src/components/layouts/layout.tsx:354]: It's my fault. Now, it works perfectly

File: src/pages/project/userSettings.tsx (hunk 3 above)
Comment (DIWAKARKASHYAP) [reply src/pages/project/userSettings.tsx:54]: I'm sorry, I don't quite understand. Could you explain a bit more?
image
i decrease little bit size of user name

**Comment by khareyash05** [comment]
Hi @marcklingen I was thinking on working on this issue. Can I please know as to why this PR was closed so that I can improve on the same? Thanks

**Pull request closed without merging**